enhancements/network/enable-flows-collection.md

```diff
@@ -0,0 +1,307 @@
+---
+title: neat-enhancement-idea
+authors:
+  - "@mariomac"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-07-29
+last-updated: 2021-07-29
+status: implementable
+see-also:
+  - "/enhancements/network/netflow.md"
+replaces:
+superseded-by:
+---
+
+# Enable flows collection
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This document defines the steps required to facilitate the enablement and collection
+of network Flows' tracing from the user side, as well as the status retrieval.
+It describes the user stories to switch on/off the flows both from the command-line
+interface and the web console, and depicts the implementation details required in
+the existing OVN-Kubernetes component, as well as the new components that should
+be created.
+
+## Motivation
+
+Flows (NetFlow, IPFIX) emission, collection and processing may have an overall
+impact in the system performance. By this reason, we need to allow users to
+switch on and off the flows' tracing feature.
+
+It is also important to allow the users to know the status of the current flows
+tracing (on or off).
+
+### Goals
```

Comment:
I think we should also add more controls in configuring flows emission, especially for fine-tuning the ipfix cache settings which can play an important role to mitigate potential issues with a too high ingestion rate.
Basically, ovs allows to control cache timeout and cache max flows that will modify the size of aggregates, hence the number of records sent to collector.

Reply:
Do you mean, making this options visible to the customer via UI or `oc`? In any case, I'm afraid that adding it here would increase too much the scope of the RFE. Maybe it's better to create another RFE with the additions to the UI as well as the changes required in the CNO to enable such configuration. What do you think?